students.ps1

```diff
@@ -96,12 +100,12 @@ try {
 
     if ($config.Environment -eq "Test")
     {
-        $OAuthUrl = https://oauthtest.presentis.nl/oauth2/token
-        $BaseUrl =  https://apitest.presentis.nl/rest/v1
+        $OAuthUrl = "https://oauthtest.presentis.nl/oauth2/token"
```

Comment:
also applies to _persons.ps1_.

Reply:
I've applied this fix to persons.ps1 as well